infra/data-pipeline.yml

```diff
@@ -718,44 +718,48 @@ tasks:
         owner: release-mgmt-analysis@mozilla.com
         source: https://github.com/mozilla/bugbug/raw/master/data-pipeline.yml
 
-#    - ID: integration-test
-#      created: {$fromNow: ''}
-#      deadline: {$fromNow: '119 hours'}
-#      expires: {$fromNow: '1 month'}
-#      provisionerId: aws-provisioner-v1
-#      workerType: relman-svc
-#      scopes:
-#        - secrets:get:project/relman/bugbug/integration
-#      payload:
-#        features:
-#          taskclusterProxy:
-#            true
-#        maxRunTime: 10800
-#        image: mozilla/bugbug-commit-retrieval:latest
-#        env:
-#          TC_SECRET_ID: project/relman/bugbug/integration
-#        command:
-#          - "/bin/bash"
-#          - "-lcx"
-#          - "git clone --quiet https://github.com/mozilla/bugbug &&
-#            cd bugbug &&
-#            git checkout ${version} &&
-#            pip install --quiet . &&
-#            pip install --quiet -r test-requirements.txt &&
-#            apt-get update &&
-#            apt-get install -y redis-server &&
-#            python -c 'import os; print(os.environ.keys())' &&
-#            bash ./scripts/integration_test.sh"
-#      metadata:
-#        name: bugbug integration test
-#        description: bugbug integration test
-#        owner: release-mgmt-analysis@mozilla.com
-#        source: https://github.com/mozilla/bugbug/raw/master/data-pipeline.yml
+    - ID: integration-test
```

Comment:
Add a comment saying this is the same as the one in .taskcluster.yml, so if we change one we don't forget to change the other